content/doc/book/updating/_pre-upgrade-wizard.adoc

```diff
@@ -0,0 +1,14 @@
+////
+This file is only meant to be included as a snippet in other documents.
+There is a version of this file for the general 'Updating Jenkins' page (index.adoc).
+This file is for the index.adoc page used in the general 'Updating Jenkins' page.
```

Comment:
```suggestion
This file is for the index.adoc page used in the general 'Updating Jenkins' page.
```
This does not appear to be included in the index page, so needs to be added there as well (if that is the idea).